src/main/java/net/querz/nbt/tag/CompoundTag.java

```diff
@@ -277,7 +282,7 @@ public int compareTo(CompoundTag o) {
 
 	@Override
 	public CompoundTag clone() {
-		CompoundTag copy = new CompoundTag();
+		CompoundTag copy = new CompoundTag((int) Math.ceil(getValue().size() * 1.33f));
```

Comment:
I meant using `size() / 0.75f` instead of `size() * 1.33f`; it has the same result respectively is more precise (since `1 / 0.75f` is actually `1.3333334f`). But combined with a comment (as suggested above) it would make the intention more clear:
```suggestion
		// Choose initial capacity based on default load factor (0.75) so all entries fit in map without resizing
		CompoundTag copy = new CompoundTag((int) Math.ceil(getValue().size() / 0.75f));
```